### ISSUE TITLE
Version 1.1.299 introduces bug when using pyxlsb.workbook.Workbook

### ISSUE THREAD

**Issue by Dr-Irv** [microsoft/pyright#4810]
**Describe the bug**
If using `pyxlsb.workbook.Workbook`, it finds the class with pyright 1.1.298, but not 1.1.299

**To Reproduce**
```python
import pyxlsb.workbook


def foo(x: pyxlsb.workbook.Workbook):
    pass
```
with 1.1.299, get error:
```text
pyright 1.1.299
pyxlsbworkbook.py
  pyxlsbworkbook.py:4:28 - error: "Workbook" is not a known member of module "pyxlsb.workbook" (reportGeneralTypeIssues)
```
With 1.1.298, get no error.

**Expected behavior**
No error. 

**VS Code extension or command-line**
command line pyright

**Additional context**

Uses `pyxlsb` version 1.0.10

This got picked up by running `pyright` on `pandas-stubs`

**Comment by erictraut** [comment]
I'm not able to repro the problem. Perhaps you have something else installed in your venv (e.g. a stub library) that is interfering with the type evaluation. Are you able to repro with a clean venv?

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release. There were a couple of issues that contributed to this bug. It manifest only for import statements with multipart module names and only if `useLibraryCodeForTypes` was disabled.

**Comment by erictraut** [comment]
This is included in pyright 1.1.300, which I just published. It will also be included in a future release of pylance.
